### PR TITLE
refactor: unify UI configuration

### PR DESCRIPTION
**Pull request by geigerzaehler**
We unify the mechanism to configure the UI. This makes it extendable if we want to add more configuration parameters.

### CODE REVIEW

File: native/index.ts [hunk 1]
```diff
@@ -352,3 +351,17 @@ function execAsync(cmd: string): Promise<{ stdout: string; stderr: string }> {
     });
   });
 }
+
+function buildConfig(): Partial<Config> {
+  const config: Partial<Config> = {};
+  if (process.env.RADICLE_UPSTREAM_UI_PROXY_ADDRESS) {
+    config.proxyAddress = process.env.RADICLE_UPSTREAM_UI_PROXY_URL;
```
Comment (rudolfs) [inline native/index.ts:358]:
```suggestion
    config.proxyAddress = process.env. RADICLE_UPSTREAM_UI_PROXY_ADDRESS;
```

**Review by rudolfs — COMMENTED** [review]
Build is broken

File: native/index.ts (hunk 1 above)
Comment (geigerzaehler) [reply native/index.ts:358]: :facepalm:

**Review by rudolfs — APPROVED** [review]
utACK

**Pull request merged**